### DISCUSSION TITLE
Dealing with N/A responses

### DISCUSSION THREAD

**Discussion by alaskamike** [nmalkin/plot-likert#34] [General]
I am trying to plot a 1 - 7 survey that has (a lot) of 0 responses. How do I remove them from the dataset? Right now I essentially have an 8 level survey.

**Comment by alaskamike** [comment]
I am trying to plot a 1 - 7 survey that has (a lot) of 0 responses. How do I remove them from the dataset? Right now I essentially have an 8 level survey.

Data:
|    | Q_01 | Q_02 | Q_03 | Q_04 | Q_05 |
|----+------+------+------+------+------|
|  0 |    0 |    0 |    0 |    0 |    0 |
|  1 |    0 |    0 |    0 |    0 |    4 |
|  2 |    4 |    2 |    3 |    5 |    3 |
|  3 |    0 |    0 |    0 |    6 |    0 |
|  4 |    0 |    0 |    0 |    0 |    0 |
|  5 |    5 |    6 |    2 |    6 |    4 |
|  6 |    1 |    1 |    1 |    4 |    1 |
|  7 |    2 |    3 |    2 |    4 |    3 |
|  8 |    2 |    2 |    3 |    4 |    3 |
|  9 |    1 |    1 |    5 |    6 |    2 |
| 10 |    6 |    6 |    5 |    6 |    5 |
| 11 |    1 |    4 |    3 |    2 |    2 |
| 12 |    5 |    3 |    2 |    5 |    2 |
| 13 |    4 |    3 |    2 |    4 |    3 |
| 14 |    1 |    1 |    1 |    4 |    1 |
| 15 |    1 |    3 |    1 |    7 |    5 |
| 16 |    0 |    0 |    0 |    0 |    0 |
| 17 |    0 |    0 |    0 |    0 |    0 |
| 18 |    5 |    5 |    4 |    2 |    3 |
| 19 |    0 |    3 |    3 |    5 |    3 |
| 20 |    0 |    0 |    0 |    0 |    4 |

`pl.plot_likert(
        df= df,
        plot_scale= pl.scales.raw7_0,
        colors= pl.colors.likert7,
        plot_percentage= True)
`

**Comment by alaskamike** [comment]
I forgot to add "drop_zeros=True," to the plot request.  plot_likert is great!

**Reply by nmalkin** [reply]
You're the one who added that argument, so all praise goes to you  :-)

**Reply by alaskamike** [reply]
Thanks, but now I'm running into another issue.  The middle value (4) is plotting to the immediate left of the center line, like it's an 8-point scale.  I'm going over the code to figure out what's going on, but any help would be appreciated.  Thanks.
